pkg/storage/storagewrappers/boundedconcurrency_test.go

```diff
@@ -0,0 +1,65 @@
+package storagewrappers
+
+import (
+	"context"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/oklog/ulid/v2"
+	"github.com/openfga/openfga/pkg/storage"
+	"github.com/openfga/openfga/pkg/storage/memory"
+	"github.com/openfga/openfga/pkg/storage/mocks"
+	"github.com/openfga/openfga/pkg/tuple"
+	"github.com/stretchr/testify/require"
+	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
+)
+
+func TestBoundedConcurrencyWrapper(t *testing.T) {
+	store := ulid.Make().String()
+	t.Logf("create a slow backend that takes 1 second per read call")
```

Comment:
This doesn't need to be logged, just leave a comment. It's noisy in the test output otherwise.